Makefile

```diff
@@ -46,9 +46,9 @@ help: ## Display this help message.
 all: run.dev
 
 clean: ## Remove all temporary files.
-	find -name '*.pyc' -delete
-	find -name '*~' -delete
-	find -name '__pycache__' -type d -delete
+	sudo find -name '*.pyc' -delete
+	sudo find -name '*~' -delete
+	sudo find -name '__pycache__' -type d -delete
```

Comment:
Do you know why we need `sudo` here?  I would prefer to make do without it, since it is inconvenient when used locally rather than as part of the tests.